inc/endpoints/class-cache-endpoint.php

```diff
@@ -51,8 +51,12 @@ public function get_route_response( $request ) {
 
 		// Ensure the action exists prior to firing.
 		if ( ! empty( $action ) ) {
+			$response = new \WP_REST_Response( true );
```

Comment:
Since, I assume, the server has a better handle on what happened with the cache than the client, perhaps the "success" string could be used as the response data directly here, and the JS then reads the string? Here might also be a more-natural place to pass the string to a translation function, which is currently missing.

Reply:
@dlh01 Definitely helpful suggestions. Thanks!